File: Java/src/com/leet_code/test/ReverseStringTest.java
package com.leet_code.test;

import com.leet_code.solution.ReverseString;

import static org.junit.Assert.*;

/**
 * Created by hu on 16/7/30.
 */
public class ReverseStringTest {

    @org.junit.Before
    public void setUp() throws Exception {

    }

    @org.junit.After
    public void tearDown() throws Exception {

    }

    @org.junit.Test
    public void testReverseString() throws Exception {
        assertEquals(ReverseString.reverseString("hello"), "olleh");
    }
}

File: C/LeetCode/tests/reverseStringTest.c
//
//  reverseStringTest.c
//  LeetCode
//
//  Created by 胡 桓铭 on 16/9/15.
//  Copyright © 2016年 胡 桓铭. All rights reserved.
//

#include "reverseStringTest.h"

TEST reverseStringEuqalGivenString(void) {
    ASSERT_STR_EQ("!olleh", reverseString("hello!"));
    PASS();
}

SUITE(reverseStringTestSuite) {
    RUN_TEST(reverseStringEuqalGivenString);
}
